docker

```diff
@@ -57,3 +57,15 @@ docker login
 
 # To push a docker image into dockerhub repository:
 docker push <image-name>:<image-tag-name>
+
+# To list all networks daemon knows
+docker network ls
+
+# To create a network
+docker network create <network name>
```

Comment:
```suggestion
# Create a specific network:
docker network create "<network_name>"
```

- Almost the same as above.
- Optional change: it's safer to use quotes.
- Optional change: it's better to be consistent: use snake-style placeholders.